packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -453,6 +455,9 @@ async function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): Pr
 }
 
 async function onDidStartDebugSessionHandler(event: vscode.DebugSession): Promise<void> {
+  if (isM365CopilotChatDebugSession(event)) {
+    void cdpClient.start();
```

Comment:
Does this correctly handle corner case like start twice?
One possible reason is that user stops then restarts debugging in a short period.

Reply:
Yes, I will fix it.